src/Ranking.elm

```diff
@@ -1,17 +1,132 @@
 module Ranking exposing (Model, Msg, init, update, view)
 
-import Debug exposing (todo)
-import Zipper exposing (Zipper)
+import Browser
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (alignLeft, alignRight, centerX, centerY, column, el, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, table, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import List.Extra exposing (count, getAt, takeWhileRight)
+import List.Zipper exposing (Zipper, after, current, isLast, next, toList, withDefault)
+import Maybe
+import SetZipper exposing (Team, createZipper)
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view >> layout []
+        , update = update
+        }
 
 
 type alias Model =
-    { teams : Zipper (Zipper Team)
+    { teams : Zipper Team
     , isShowingInfo : Bool
+    , inPage : Int
     }
 
 
-type alias Team =
-    { number : Int
-    , name : String
-    , score : Float
-    }
+type Msg
+    = NextPage
+
+
+moveZipperBy : Int -> Zipper a -> Zipper a
+moveZipperBy moveBy zipper =
+    if moveBy == 0 || isLast zipper then
+        zipper
+
+    else
+        moveZipperBy (moveBy - 1) <| withDefault (current zipper) <| next zipper
+
+
+arrangementThePage : List Team -> Element.Element Msg
+arrangementThePage listOfTeams =
+    table []
+        { data = listOfTeams
+        , columns =
+            [ { header = Element.text "Number"
+              , width = fill
+              , view =
+                    \team ->
+                        Element.text <| String.fromInt team.number
+              }
+            , { header = Element.text "Score"
+              , width = fill
+              , view =
+                    \team ->
+                        Element.text <| String.fromInt team.score
+              }
+            , { header = Element.text "Name"
+              , width = fill
+              , view =
+                    \team ->
+                        Element.text team.name
+              }
+            ]
+        }
+
+
+getNeededList : Int -> Zipper a -> List a
+getNeededList neededInPage zipper =
+    List.take neededInPage <| current zipper :: after zipper
+
+
+teamToString : Team -> String
+teamToString team =
+    String.fromInt team.number
+        ++ " "
+        ++ team.name
+        ++ " "
+        ++ String.fromInt team.score
```

Comment:
The name of the function does not expand on the information given by the type - `Team -> String` literally reads "team to string".
When that's the case, it usually means your name can improve or your types are spot-on and the name can be much shorter.

Either way, I'd use `String.concat` instead of multiple `++`s and destructure `team` at the declaration.